chia/wallet/coin_selection.py

```diff
@@ -97,7 +102,7 @@ async def select_coins(
                 coin_set = {greater_coin}
         return coin_set
     else:
-        # if smaller_coin_sum == amount and len(smaller_coins) >= max_num_coins.
+        # if smaller_coin_sum == amount and (len(smaller_coins) >= max_num_coins or amount == 0)

```

Comment:
what is the reason for leaving this comment in?

Reply:
This improves readability, and makes it easier to understand, Mariano suggested the comment so I think it's a good idea.